eslint/babel-eslint-parser/test/index.js

```diff
@@ -22,8 +22,9 @@ const isESLint7 = ESLINT_VERSION.startsWith("7.");
 const dirname = path.dirname(fileURLToPath(import.meta.url));
 
 // @babel/eslint-parser 8 will drop ESLint 7 support
-const itESLint7 = isESLint7 && !process.env.BABEL_8_BREAKING ? it : it.skip;
-const itESLint8 = isESLint7 ? it.skip : it;
+const dummy = () => {};
```

Comment:
```suggestion
const dummy = () => {};
dummy.skip = dummy.only = dummy;
```

So that when during local development we temporarily `.skip` a test, it can also work for `itESLint7`/`itESLint8`.